components/track/TrackMenu.vue

```diff
@@ -62,7 +64,20 @@ const dropdownMenuItemsForTrack = (track: TrackModel) => {
   items.push({
     icon: "icon.person",
     text: t("track.dropdown.go-to-contributors"),
-    link: { name: "browse" }, // TODO: change link
+    clickFunction: () => {
+      if (track.contributors && track.contributors?.length > 1) {
+        showContributorsList.value = true;
+      } else if (
+        track.contributors &&
+        track.contributors.length === 1 &&
+        track.contributors[0]?.id
+      ) {
+        navigateTo(`/playlist/contributor/${track.contributors[0].id}`);
+        if (showMediaPlayer) {
```

Comment:
why does clicking on a contributor affect the player?

Reply:
Because the intention when clicking on the contributor is to view the profile of the contributor, I assume, having the player open then covers up the screen so you have to minimize it manually. I just do it automatically which I thought is more expected behaviour but I might be wrong?

Reply:
The idea is that the player claims the right side of view and is not on top of anything. I believe on smaller screen sizes it does cover some content since the content has a minimum width.

Anyway, I don't think we should hide the player

Reply:
Done